srtcore/group.cpp

```diff
@@ -1447,9 +1447,9 @@ int CUDTGroup::sendBroadcast(const char* buf, int len, SRT_MSGCTRL& w_mc)
     // Links that were successful, have the len value in state.
 
     // First thing then, find out if at least one link was successful.
-    // This might even be one of the idlers only, this doesn't matter.
+    // This might even be one of the idle links only, this doesn't matter.
     // If there were any running links successful, they have set the sequence.
-    // If there were only some reactivated idlers successful, the first
+    // If there were only some successfully reactivated idle link, the first
```

Comment:
This changes the meaning of the comment. It's not about links being successfully reactivated, but that the sending operation through that link was successful (the comment in the first line speaks about the same thing).

Reply:
This comment is very unclear. How about this way?

```c++
// First thing then, find out if at least one link was successful.
// even if it was one of the idle links.
// The first successful link sets the sequence.
```

Reply:
Yeah, it's better and more concise, although it skipped one important information worth highlighting: if the successful link was one of so far active links, and there was also an activated idle link beside this, this idle link derives the sequence from the active link, while when there were no active links (only idle), then the sending sequence number was taken as a good deal as is from the FIRST activated idle link, and if any other idle links were activated as well (although I don't think such a situation would be possible, but in theory), they will derive this sequence. It's important to understand that all links - just like with broadcast - must use perfectly synchronized sequence numbers, although the "pattern for others to adjust to" is the value from the currently active links (all of them should be also in sync already between one another).